src/components/Playlists/Playlist.svelte

```diff
@@ -83,12 +83,17 @@
 
 			for (let index = 0; index < ownersIds.length; index++) {
 				const element = ownersIds[index];
-
 				let playerService = createPlayerService();
-				let owner = await playerService.fetchPlayerOrGetFromCache(element);
-
+				let owner = playerService.fetchPlayerOrGetFromCache(element);
 				if (owner?.playerId != playerId) {
-					canModify = false;
+					if (playerService.isMainPlayer(owner.playerId)) {
```

Comment:
The cause of this issue is that I added `keepOriginalId=true` to the `fetchPlayer` request. So the player fetched by the old ID has it as a `playerId` property. Let's make it a param and set it here to `false`, so it will return a new ID.